java/src/main/java/com/logicalclocks/hsfs/FeatureGroup.java

```diff
@@ -173,7 +172,7 @@ public Dataset<Row> read(String wallclockTime, Map<String, String> readOptions)
    */
   public Dataset<Row> readChanges(String wallclockStartTime, String wallclockEndTime)
```

Comment:
Why isnt it defined in the base class? (multiple methods like this here)

Reply:
because it is deprecated. should we define deprecated methods in base class?

Reply:
proper deprecation annotations are set in other (java doc) PR